cmd/sealos/cmd/inspect.go

```diff
@@ -0,0 +1,83 @@
+/*
+Copyright 2022 chenchuanle6@gmail.com.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	bb "github.com/labring/sealos/pkg/image/buildah/cluster/buildah"
+	"github.com/labring/sealos/pkg/utils/exec"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"golang.org/x/term"
+)
+
+var exampleInspect = `
+  sealos inspect imageName
+  sealos inspect containerName
```

Comment:
containerName->clusterName, container is cluster, distinguish with docker container.

Reply:
Is clusterName specified by the  --name parameter of `sealos run` command?